action.yml

```diff
@@ -16,6 +16,14 @@ inputs:
     description: "Path of the directory containing your site"
     required: false
     default: "."
+  deno-version:
+    description: "The Deno version to use when deno.json/deno.jsonc/deno.lock exists"
+    required: false
+    default: "1.x"
+  deno-setup-task:
+    description: "Name of the optional Deno task to run before build (will be skipped if undefined)"
+    required: false
+    default: "setup"
```

Comment:
Could you explain why this is needed? Can people not use `prebuild` scrips and that kind of thing? I’m hesitant to add a Deno-only configuration option for something like this.